app/models/validations/sales/financial_validations.rb

```diff
@@ -36,12 +36,8 @@ def validate_combined_income(record)
     end
   end
 
-  def validate_cash_discount(record)
-    return unless record.cashdis
-
-    unless record.cashdis.between?(0, 999_999)
-      record.errors.add :cashdis, I18n.t("validations.financial.cash_discount_invalid")
-    end
+  def validate_monthly_leasehold_charges(record)
+    record.errors.add :mscharge, I18n.t("validations.financial.monthly_leasehold_charges.not_zero") if record.mscharge&.zero?
```

Comment:
Is this so that the validation message is different from min/max?

Reply:
exactly